main.js

```diff
@@ -1,17 +1,21 @@
-let addToDoButton = document.getElementById('buttonAddTask');
-let inputTask = document.getElementById('inputAddTask');
-let toDoesContainer = document.getElementById('toDoesContainer');
+const btnAdd = document.querySelector('.btn');
+const input = document.querySelector('.input');
+const toDos = document.querySelector('.toDos');
 
-addToDoButton.addEventListener('click', function(){
-    var paragraph = document.createElement('p');
-    paragraph.innerText = inputTask.value;
-    toDoesContainer.appendChild(paragraph);
-    inputTask.value="";
-    paragraph.addEventListener('click', function(){
-        paragraph.style.textDecoration = "line-through";
-    })
-    paragraph.addEventListener('dblclick', function(){
-        toDoesContainer.removeChild(paragraph);
+btnAdd.addEventListener('click', (e) => {
+    const todo = document.createElement('p');
+    todo.innerText = input.value;
+    toDos.appendChild(todo);
+    input.value= "";
```

Comment:
Let's work on formatting code a bit. 
```suggestion
    input.value = "";
```